src/PluginClient.php

```diff
@@ -62,6 +62,36 @@ public function __construct($client, array $plugins = [], array $options = [])
     }
 
     /**
+     * Adds a plugin to the client.
+     *
+     * @param Plugin $plugin
+     */
+    public function addPlugin(Plugin $plugin)
```

Comment:
:+1: Agree, this behavior is crucial to be mentioned as the order of plugins matter. Probably this is a reason against allowing to add plugins runtime: it can cause confusions.